packages/blade/src/components/Box/BaseBox/types/propsTypes.ts

```diff
@@ -174,6 +175,8 @@ type BaseBoxVisualProps = MakeObjectResponsive<
       | BackgroundColorString<'feedback'>
       | BackgroundColorString<'surface'>
       | BackgroundColorString<'interactive'>
+      | BackgroundColorString<'overlay'>
+      | BackgroundColorString<'popup'>
       | 'transparent';
     // | (string & Record<never, never>);
```

Comment:
Removing this means we don't accept string colors on box. Is that intentional now @saurabhdaware @anuraghazra